### ISSUE TITLE
Problems on Windows 10 Home 

### ISSUE THREAD

**Issue by znmeb** [hackoregon/backend-examplar-2018#50]
@mxmoss has run into an error starting the API container on a Windows 10 Home / Docker Toolbox configuration. @nam20485 and I have been able to reproduce it with Docker for Windows from WSL Ubuntu, but that's not a supported Docker use case. I can reproduce it on this repo with the sample "dead_songs" dataset, and I'm going to attempt to troubleshoot it with Docker Machine on Linux,

Meanwhile, on Docker for Windows on Windows 10 Pro, both repos work if you do the build and start from PowerShell.

**Comment by bhgrant8** [comment]
thank you for documenting, let us know if further help is needed.

**Comment by znmeb** [comment]
I've got some clues finally. Docker Toolbox (what @mxmoss and other Windows < Window 10 Pro users have) uses Docker Machine to create and manage the VirtualBox machines on Windows. It turns out that all of this magic also works on Linux hosts, which nobody uses because Docker is native for Linux - no virtual machine required.

So I installed Docker Machine and VirtualBox on Linux, created a virtual machine and logged into it. Then I cloned this repo and tried to run `bin/build.sh -d`. It croaked - it seems the Linux virtual machine doesn't have `bash` - just `sh`! It's based on `busybox`, a rather stripped "Linux-like" OS that provides just enough command line to make you think you've got a real system.

So as long as the scripts will work using just `sh` command, it should work - I'm testing that now.

**Comment by nam20485** [comment]
Put me on the PR for review please. I'd like to try testing it when you have something.

**Comment by nam20485** [comment]
So is the fix for this issue to change the "shebang" lines in the scripts from `#!bin/bash` to `#!bin/sh`?

**Comment by znmeb** [comment]
@nam20485 Not really - it turns out the problem is probably a silent failure of Docker mounting the host repository into `/code/ in the container. :-(

**Comment by nam20485** [comment]
Fair enough. Ignore my previous comments, I was confused with old info but I think I am up to speed now.

**Comment by znmeb** [comment]
A bit of an update - I spent a few hours yesterday testing on Windows 10 *Pro* and was unable to find a simple, repeatable way to get this task done. So I've gone nuclear ;-)

Seriously, what I did was create a new Dockerfile / compose file / script combo that doesn't mount any host files into a container. Instead it copies the inputs to the image and copies the resulting code back out to the repo. 

And it's a one-step process - you edit the environment variables, copy your backup file in and run a script. When it's done you have the database image running in its container, the API image running in it's container with the scaffolding code. When you terminate it, the script copies the code to your repo.

**Comment by znmeb** [comment]
Fix is in pull request #49
